### DISCUSSION TITLE
Noob question！Is there anyway I can manually add same tag to multi files?

### DISCUSSION THREAD

**Discussion by Mirrors894** [Difegue/LANraragi#834] [Q&A] [answered]
The default EH tag plugin doesnt work for me. They will add wrong tags and change title. So I prefer add tag manually.
For exmaple, I wanna add "artist: xxx" this tag to 40 files. What should I do? (I already put them in subfolder, but I also need tag to search)

**Comment by Difegue** [comment, marked as answer]
👋 I'd recommend using the "Tag Copier" plugin in batch tagging for this.

**Reply by Mirrors894** [reply]
Thank you for your response. I just found out that the problem is that only alphabet and numbers are accepted, and other languages such as Japanese (kanji, katakana) are not available? If I try to add kanji tags with tag copier, the bar will show (code 1006).
But now I have a new question, how to batch delete a tag from multiple files?

**Reply by Difegue** [reply]
There's nothing for batch deletion at the moment sadly -- It's something I should eventually look into adding to the plugin system.  

As for 1006, do you also get it if running the plugin manually on a single file? I don't know if it's a batch tagging/websocket issue or a server one. (likely websockets tbh)

**Reply by Mirrors894** [reply]
I use tag copier and only select one file try to add tag “[鳥山明]” , still shows 1006. (Batch Operations-Use plugin :tag copier- Override Plugin Global Arguments-add [鳥山明]-Start Task-code 1006)
If I add manually on single file (admin options-edit Archive metadata), It works. I think its the problem of tag copier that only support alphabet and numbers.

**Reply by Difegue** [reply]
Thanks for the report - There's a bug in batch tagging where it can't accept utf8 characters, I'll have that fixed in 0.9.0.
